src/app/shared/case-editor/case-edit-page.html

```diff
@@ -1,3 +1,4 @@
+<a (click)="previous()" href="javascript:void(0)" class="back">&lt; Back</a>
```

Comment:
The control ```[disabled]="!hasPrevious()" ``` seems to be missing here. If there is no previous page, this link should be `hidden` or `disabled`

Reply:
The Back link is now reverted back to Previous button.   The comment is incorporated.